types/styled-system/index.d.ts

```diff
@@ -0,0 +1,603 @@
+// Type definitions for styled-system 2.3
+// Project: https://github.com/jxnblk/styled-system#readme
+// Definitions by: Marshall Bowers <https://github.com/maxdeviant>
+//                 Ben McCormick   <https://github.com/phobon>
+//                 Justin Bennett  <https://github.com/zephraph>
+//                 Christopher Pappas <https://github.com/damassi>
+//                 Eloy Durán      <https://github.com/alloy>
```

Comment:
@maxdeviant @phobon FYI You are listed as maintainers here because you’ve done work on these types, but it does mean that the DT bot will ping you and ask for a review anytime a PR for this file is made. Please let us know if you’d like to be removed, for now we’ll assume you want to be listed.

Reply:
Yeah, I'm absolutely happy with this, thanks for kicking this all off 👍